src/features/user/ManagePayouts/components/BankDetailsForm.tsx

```diff
@@ -171,6 +175,25 @@ const BankDetailsForm = ({
               />
             )}
           />
+          <Controller
+            name="bankCountry"
+            control={control}
+            render={({ field: { onChange, value, name } }) => (
+              <AutoCompleteCountry
+                label={t('labels.bankCountry')}
+                name={name}
+                defaultValue={
+                  value ||
+                  (getStoredConfig('loc').countryCode === 'T1' ||
```

Comment:
Functionality is fine but why haven't we used `getStoredConfig('country')` here instead?